src/components/mainMenu.js

```diff
@@ -0,0 +1,21 @@
+import mainMenuButton from "./mainMenuButton";
+
+const mainMenu = (callback) => {
+    const section = document.createElement('div');
+    section.classList.add('mainMenu');
+
+    const peopleButton = mainMenuButton('mainMenu__button','People',callback)
```

Comment:
Tutaj w tej linijce proponuję jeszcze zmienić "People" na "Characters" bo niestety taki API przyjmuje ciąg, żeby dać nam dane o osobach i dzięki temu będzie można dużo łatwiej to później obsłużyć :)

Reply:
Jasne!